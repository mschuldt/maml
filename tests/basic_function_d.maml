arduino = Arduino(desktop=True)

@function
def ftest():
    print_i(123)
    print_s("test ok!")

@block(once)
def test():
    print_s("calling ftest:")
    ftest()
    print_s("Done.")
    die(0)
    
arduino.send(ftest)
arduino.send(test)
