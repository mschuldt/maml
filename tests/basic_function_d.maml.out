Initializing avm...
Ready.

calling ftest:
123
test ok!
Done.
bye.
